manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -152,6 +152,26 @@ metadata:
   name: internal-feature-states.csi.vsphere.vmware.com
   namespace: vmware-system-csi
 ---
+apiVersion: v1
```

Comment:
I guess the last time we moved `Service` definition above `Deployment` created a problem in the pipelines. Is that prevented here?

Reply:
I have verified deploying the driver using the updated YAML file. Moving service up before deployment should not cause any issue. We need to fix the pipeline if we hit any issues.

Reply:
Refer: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1196

Reply:
We have reverted the change before since pipelines fail. Can we run the pipelines for this PR and validate the same